magenta/interfaces/midi/midi.py

```diff
@@ -247,16 +262,21 @@ class MonoMidiPlayer(threading.Thread):
     _outport: The Mido port for sending messages.
     _sequence: The monohponic, chronologically sorted NoteSequence to play.
     _stop_playback: A boolean specifying whether the playback should stop.
+    _bpm: The tempo in beats per minute. Copied from input sequence.
   Args:
     outport: The Mido port for sending messages.
     sequence: The monohponic, chronologically sorted NoteSequence to play.
   """
   daemon = True
 
-  def __init__(self, outport, sequence):
+  def __init__(self, outport, sequence, metronome_velocity):
     self._outport = outport
     self._sequence = sequence
     self._stop_playback = False
+    if len(sequence.tempos) == 1:
```

Comment:
Raise an exception if there isn't a single tempo. Make sure to document that exception in the docstring for the class.